src/output.cpp

```diff
@@ -134,13 +134,13 @@ lame_t airlame_init(mix_modes mixmode, int highpass, int lowpass) {
 		return NULL;
 	}
 
-	lame_set_in_samplerate(lame, WAVE_RATE);
-	lame_set_VBR(lame, vbr_mtrh);
-	lame_set_brate(lame, 16);
-	lame_set_quality(lame, 7);
-	lame_set_lowpassfreq(lame, lowpass);
-	lame_set_highpassfreq(lame, highpass);
-	lame_set_out_samplerate(lame, MP3_RATE);
+    lame_set_in_samplerate(lame, WAVE_RATE);
+    lame_set_VBR(lame, vbr_off);
```

Comment:
This may be undesirable for continuous recordings, particularly with large periods of silence

Reply:
Tested with this change alone (disabling VBR) and a 4m30s recording of silence (channel with no activity and configured as `continuous = true`) increased in size from 265K to 528K